requirements.in

```
@@ -0,0 +1,8 @@
+#-c constraints.in # <-- contains known limitations
+
+# staticjinja generates html from templates
+staticjinja>=4.1.3
+# pyyaml loads page data from yaml files
+pyyaml
+# libsass compiles sass files to css
+libsass
```

I recommend putting comments on the same line whenever possible. This also applies to normal Python modules. The reason is that new lines may be accidentally added over time, between a comment and the line it refers to. If that happens, it may go unnoticed, and at some point, the comment line might be above some other dependency and wouldn't make sense to the readers. Such cases may need a git paleontology effort to figure out what a comment means, later on. This happens because humans often ignore comments when editing the source code, being focused on something else.
But being on the same line, there's a higher chance that a comment would be noticed and synchronized with the code changes there. Plus, it won't be detached and won't drift to some disconnected part of the file over time.
```suggestion
staticjinja >= 4.1.3  # generates html from templates
pyyaml  # loads page data from yaml files
libsass  # compiles sass files to css
```